core/src/main/java/org/apache/calcite/sql/SqlUtil.java

```diff
@@ -304,7 +304,12 @@ public static void unparseFunctionSyntax(SqlOperator operator,
       }
       SqlIdentifier id = function.getSqlIdentifier();
       if (id == null) {
-        writer.keyword(operator.getName());
+        if (((SqlFunction) operator).getFunctionType() == SqlFunctionCategory.USER_DEFINED_FUNCTION
```

Comment:
Can you please add a simple comment over here as it only works, when we translate SQL from Mig.

Reply:
done